examples/enable-persistence-queue/enable-persistence-queue-values.yaml

```diff
@@ -0,0 +1,29 @@
+clusterName: CHANGEME
+splunkPlatform:
+  endpoint: CHANGEME
+  token: CHANGEME
+  index: CHANGEME
+  metricsIndex: CHANGEME
+  metricsEnabled: true
+  tracesEnabled: true
+  logsEnabled: true
+  sendingQueue:
+    persistentQueueEnabled:
+      logs: true
+      metrics: true
+      traces: true
+      storagePath: "/var/addon/splunk/persist"
+
+logsEngine: otel
+
+agent:
+  securityContext:
```

Comment:
Is the securityContext required?
- If yes, please add a note similiar to [this](https://github.com/signalfx/splunk-otel-collector-chart/blame/08be4e5853a7e3d9ce1d42ec5d755a68ecca53e1/docs/advanced-configuration.md#L526) in your newly added advanced-configuration.md documentation.
- If no, I d remove the securityContext here or leave a note it is optional.